edxlearndot/models.py

```diff
@@ -38,7 +38,7 @@ class EnrolmentStatusLog(models.Model):
         help_text="The last status sent to Learndot."
     )
 
-    class Meta(object):
+    class Meta(object):  # pylint: disable=useless-object-inheritance
```

Comment:
nit: python3 classes all inherit from object:
```suggestion
    class Meta:
```